packages/backend/src/database/migrations/20230530130533_validations_update_table.ts

```diff
@@ -0,0 +1,25 @@
+import { Knex } from 'knex';
+
+const ValidationTableName = 'validations';
+
+export async function up(knex: Knex): Promise<void> {
+    await knex.schema.alterTable(ValidationTableName, (tableBuilder) => {
+        tableBuilder.string('error_type').nullable().defaultTo(null);
```

Comment:
I don't think this defaultTo(null) is necessary, check if we do it on other migrations

Reply:
exactly, we don't! thank you